pkg/etcdcli/etcdcli.go

```diff
@@ -282,6 +293,8 @@ func (g *etcdClientGetter) Status(ctx context.Context, clientURL string) (*clien
 	}
 
 	defer g.clientPool.Return(cli)
+	ctx, cancel := context.WithTimeout(ctx, DefragDialTimeout)
```

Comment:
I think that's fine and can be 30s, the actual defragmentation call could take longer. Can you change that back to `DefaultCliTimeout` please?

Reply:
ok， i will change it back to DefaultClientTimeout